### PR TITLE
formatting changes

### PR DESCRIPTION
**Pull request by NatalieEO**
Portion of dockstore/dockstore#2964 I think we discussed in December no longer pointing out private tools or getting rid of them since it's not really used and the order of facets tbd once we work out other changes later (like the new facet ideas). 
Screen Shot 2020-01-13 at 1 16 38 PM
Screen Shot 2020-01-13 at 1 16 45 PM

### CODE REVIEW

**Comment by denis-yuen** [comment]
Pending build success

**Comment by garyluu** [comment]
The alignment looks odd:

image

File: src/app/search/search.component.html [hunk 1]
```diff
@@ -55,34 +58,28 @@
...
               </div>
               <div class="search-facet-right">
-                <p>({{ orderedBuckets.get(key).SelectedItems.get(subBucket) }})</p>
+                <p>{{ orderedBuckets.get(key).SelectedItems.get(subBucket) }}</p>
               </div>
             </div>
           </div>
           <div *ngFor="let subBucket of getKeys(orderedBuckets.get(key).Items); let i = index">
-            <div class="panel-container-label">
+            <div class="panel-container-label" fxLayoutAlign="space-between center">
```
Comment (garyluu) [inline src/app/search/search.component.html]: FlexLayoutModule isn't even imported in this component

**Comment by NatalieEO** [comment]
> The alignment looks odd:
> 
> image

**Comment by NatalieEO** [comment]
Screen Shot 2020-01-14 at 1 49 57 PM

**Pull request merged**